shared/oae/css/oae.base.css

```diff
@@ -897,3 +897,32 @@ ul.as-list li.as-result-item.active {
 .oae-markdown-embedded p:first-child {
     margin-top: 0;
 }
+
+/**
+ * Markdown table styles
+ */
+
+.oae-markdown-embedded table {
+    margin: 20px 0;
+    border-bottom: 1px solid #ECECEC;
```

Comment:
We try to avoid defining colors in the base CSS file because of skinning reasons. I've left a question for Sam to check if these should be hardcoded or driven by skin config values.